.github/workflows/bitcoin-tests.yml

```diff
@@ -140,6 +140,11 @@ jobs:
         with:
           btc-version: "25.0"
 
+      ## Increase open file descriptors limit
+      - name: Increase Open File Descriptors
+        run: |
+          sudo prlimit --nofile=4096:4096
```

Comment:
hell yes!

Reply:
🤞

Reply:
Doesn't look promising for these tests, but curious if it would help your network tests where you know it is a problem.